src/ProfileEndpoints.jl

```diff
@@ -246,11 +271,39 @@ end
 
 end  # if isdefined
 
+###
+### Debug super-endpoint
+###
+
+function debug_profile_endpoint_with_stage_path(stage_path = nothing)
+    stage_path = stage_path === nothing ? tempdir() : stage_path
+    function debug_profile_endpoint(req::HTTP.Request)
+        uri = HTTP.URI(req.target)
+        qp = HTTP.queryparams(uri)
+        if isempty(qp)
+            @info "TODO: interactive HTML input page"
+            return HTTP.Response(400, cpu_profile_error_message())
+        end
+        req_type = get(qp, "type", "profile")
```

Comment:
I might also make this a longer name to avoid any potential conflict if a future debugging profile type itself has a param called type. Maybe `debug_type` or `profile_type` or something? Not 100% sure tho

Reply:
Replaced with `profile_type`.